.github/workflows/deploy-apps.yml

```diff
@@ -41,6 +45,17 @@ jobs:
           scope: ${{ secrets.VERCEL_SCOPE }}
           working-directory: packages/app/dist
           github-comment: false
+      - uses: amondnet/vercel-action@v19.0.1+1
+        id: vercel-deployment-app-production
+        if: github.event_name == 'push' ||  github.event_name == 'release'
+        name: Deploy authenticator app to production
+        with:
+          vercel-token: ${{ secrets.VERCEL_TOKEN }}
+          vercel-org-id: ${{ secrets.ORG_ID }}
+          vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID_APP }}
+          vercel-args: '--prod'
+          scope: ${{ secrets.VERCEL_SCOPE }}
+          working-directory: packages/app/dist
```

Comment:
I was thinking about this the other day. I think a better location for this step would be in the publish job, so it runs after lerna version has bumped the version. Otherwise looks good.

Reply:
Ok yeah, that makes sense!